Source/SpectrumAnalysis/KTSequentialTrackFinder.cc

```diff
@@ -329,225 +341,312 @@ namespace Katydid
+                                this->EmitPreCandidate(*lineIt);
+                            }
+                        }
+                        // in any case, this line should be removed from the vector with active lines
+                        lineIt = fActiveLines.erase(lineIt);
+                    }
+                    else
+                    {
+                        // Under these conditions a point will be added to a line
+                        bool timeCondition = tempPoint.fTimeInRunC > lineIt->GetEndTimeInRunC();
+                        bool anyPointCondition = std::abs(tempPoint.fFrequency - (lineIt->GetEndFrequency() + lineIt->GetSlope()*(pointIt->fTimeInAcq - lineIt->GetEndTimeInAcq()))) < fFrequencyAcceptance;
+                        bool secondPointCondition = std::abs(tempPoint.fFrequency - (lineIt->GetEndFrequency() + lineIt->GetSlope()*(pointIt->fTimeInAcq - lineIt->GetEndTimeInAcq()))) < fInitialFrequencyAcceptance;
+
+                        // if point matches this line: insert
+                        if (timeCondition and anyPointCondition)
+                        {
+                            lineIt->AddPoint(tempPoint);
+                            (this->*fCalcSlope)(*lineIt);
+                            match = true;
+                            break;
+                        }
+                        // if this line consists of only one point so far, try again with different radius
+                        else if (lineIt->GetNPoints() == 1 and timeCondition and secondPointCondition)
+                        {
+                            KTDEBUG(stflog, "Trying initial-frequency-acceptance "<<fInitialFrequencyAcceptance);
+                            lineIt->AddPoint(tempPoint);
+                            (this->*fCalcSlope)(*lineIt);
+                            match = true;
+                            break;
+                        }
+                        // if not try next line
+                        else
+                        {
+                            ++lineIt;
+                        }
+                    }
+                }//end of while loop. all existing lines were compared
+
+                // if point was not picked up
+                if (match == false)
+                {
+                    //KTDEBUG(stflog, "Starting new line");
+
+                    KTSequentialLineData newLine;
+                    newLine.SetSlope( fInitialSlope );
+                    newLine.SetAcquisitionID( acqID );
+                    newLine.SetComponent( component );
+                    newLine.AddPoint(tempPoint);
+                    (this->*fCalcSlope)(newLine);
+                    fActiveLines.push_back(newLine);
+                    match = true;
+                }
+            }
+        }
+        return true;
+    }
+
+    bool KTSequentialTrackFinder::LoopOverHighPowerPoints(STFDiscriminatedPowerSortedPoints& points, uint64_t acqID, unsigned component)
```

Comment:
Did you make changes to this method, or was it just moved in a way that causes the lines to be shown as newly added?

Reply:
Interesting.  There were some small changes I made, like adding some debug lines that are commented out.  But I didn't do anything substantial.